.github/workflows/build.yml

```diff
@@ -337,6 +337,26 @@ jobs:
   #        name: macos-build
   #        path: |
   #          app/build/macos/Build/Products/Release/butterfly.app
+  build-snap:
+    name: Build Snap
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        architecture:
+          - amd64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt@sha256:df15403e06a03c2f461c1f7938b171fda34a5849eb63a70e2a2109ed5a778bde
```

Comment:
What's this image?

Reply:
https://github.com/diddlesnaps/snapcraft-multiarch-action

Reply:
copied from gimp snap.
https://github.com/snapcrafters/gimp/blob/master/.github/workflows/build-prs.yml